pkg/operator/ceph/config/conditions.go

```diff
@@ -44,46 +45,48 @@ func ConditionExport(context *clusterd.Context, namespace, name string, conditio
 
 // setCondition updates the conditions of the cluster custom resource
 func setCondition(context *clusterd.Context, namespace, name string, newCondition cephv1.Condition) {
```

Comment:
Would it be too verbose to write the condition and message to the log every time this method is called? This would make the operator log more readable to see what stage the operator is in.

Reply:
It's already logging that.

Reply:
Also to show the condition and message everytime can confuse a user right? Right now I am logging whenever 
- New Condition is set
- Status of the existing condition changed (True to False or False to True)
- There is a change in the message of the existing condition

Reply:
The log messages don't need to match only to when the state changes. The log messages can show a sequence of what the operator is doing, which is helpful for debugging.  If it is helpful for basic control flow, maybe it would be helpful to always log it. If it is too verbose then it could be at Debug level. I'd be curious to see an example operator log with it set to Info level and see how useful it would be.